app/assets/javascripts/modules/search-highlight.es6

```diff
@@ -0,0 +1,29 @@
+{
+  'use strict';
+
+  class SearchHighlight {
+    start(el) {
+      const term = this.getUrlParameter(encodeURIComponent('search[q]')),
+      regEx = new RegExp(`(${term})`, 'gi');
+
+      if (!term) {
+        return;
+      }
+
+      $(el).find('.js-allow-highlighting').each((index, td) => {
+        $(td).html($(td).html().replace(regEx, '<mark>$1</mark>'));
+      });
+    }
+
+    getUrlParameter(name) {
+      name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+
+      const regex = new RegExp(`[\\?&]${name}=([^&#]*)`),
+      results = regex.exec(location.search);
```

Comment:
I know we discussed this at the beginning, but I really don't like this declaration style. Indenting the variables below the first `const` would make things so much more readable. 

Perhaps we can address this in another PR. I think we should also revisit the file naming convention to match the class they return, and make consistent how we deal with `$variables`.